README.rdoc

```diff
@@ -1,12 +1,18 @@
+{<img src="https://travis-ci.org/sul-dlss/checksum-tools.svg?branch=master" alt="Build Status" />}[https://travis-ci.org/sul-dlss/checksum-tools]
+
```

Comment:
Since this is a quick-fix to a deprecated gem, I am trying to not to get too entrenched in adding a huge suite of tools.

I needed to work out the Travis build (and tell Travis to use ruby 2.3.1, so it made sense to add the Travis badge while I was there).

I'm sure there is lots of love that could happen to this particular gem (i.e. rubocop) but it seemed a bit out of scope for the work we were being asked to accomplish.